sky/backends/provision_utils.py

```diff
@@ -177,6 +177,7 @@ def bulk_provision(
     bootstrap_config = provision_common.InstanceConfig(
         provider_config=original_config['provider'],
         authentication_config=original_config['auth'],
+        docker_config=original_config.get('docker', {}),
```

Comment:
We need to manually extract `docker_login_config` and put it into `docker_config` here... Reference: https://github.com/skypilot-org/skypilot/blob/7d59ff9b4c2306c85ef61aa8026a0f2a24b8608c/sky/skylet/providers/gcp/node_provider.py#L390-L394